examples/plugins/windows-plugin/run.ps1

```diff
@@ -0,0 +1,6 @@
+$strString = "Hello World"
+$outputFile = 'c:\tmp\results\file.txt'
+$doneFile = 'c:\tmp\results\done'
+
+$strString | Out-File $outputFile -Encoding ascii
+$outputFile | Out-File $doneFile -Encoding ascii
```

Comment:
I'd like something more impressive than "hello world". Open to ideas, would prefer something clearly Windows-ish

Reply:
I came across this powershell command [`Get-WinEvent`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.diagnostics/get-winevent?view=powershell-7) which gets events from the event logs. I tried it out on my Windows partition with the following `Get-WinEvent -LogName System` but that seems to include historical events that were stored, not just since the machine was booted. There are some other examples on that page though which you could probably modify and use for this.